src/actions/ghActions.js

```diff
@@ -0,0 +1,22 @@
+// import axios from 'axios';
+
+import {
+  LOADING,
+  LOADED,
+  // ERROR,
+  // AUTHOR_PROFILE
+} from './types';
+
+// import isEmpty from './../utils/validate';
+
+export const loading = () => {
```

Comment:
```suggestion
export const loading = () => ({
     type: <any_data>,
})
```

Reduce lot of lines and looks clean :sweat_smile: